Model/Sync/Customers/Processor.php

```diff
@@ -157,25 +157,23 @@ public function process($retryCustomers = [], $storeIds = [])
      */
     public function processCustomer($customer, $customerAddress = null)
     {
-        $customerAccountShared = $this->config->isCustomerAccountShared();
+        $isCustomerAccountShared = $this->config->isCustomerAccountShared();
         /** @phpstan-ignore-next-line */
         foreach ($this->config->getAllStoreIds(false) as $storeId) {
             $this->emulateFrontendArea($storeId);
-            if (!$this->config->isCustomerSyncActive() || (
```

Comment:
what about the isActive check?

Reply:
Hey @EranAvidor, this method `processCustomer` is being referenced in 2 observers:
![image](https://user-images.githubusercontent.com/10895971/158056938-ae65b05b-2676-4697-975b-795e3ffe33ed.png)
I removed this isActive check from here intentionally, as it is already being checked in earlier in the caller methods :).